src/components/QuoteButton.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable react/button-has-type */
+import React, { useState } from 'react';
```

Comment:
- [ ] Kindly make sure you fix your linters errors from your code base instead of disabling them, this for clean code, best practices reasons and as required for all projects. I think you can fix them by just adding the `type` attribute in each `button` tag(e.g.: `type="button",...` )